pylot/perception/detection/efficientdet_operator.py

```diff
@@ -78,17 +78,17 @@ def load_serving_model(self, model_name, model_path, gpu_memory_fraction):
         detection_graph = tf.Graph()
         with detection_graph.as_default():
             # Load a frozen graph.
-            graph_def = tf.GraphDef()
-            with tf.gfile.GFile(model_path, 'rb') as f:
+            graph_def = tf.compat.v1.GraphDef()
+            with tf.io.gfile.GFile(model_path, 'rb') as f:
```

Comment:
Can you also add a comment or TODO here to remove the `EfficientDetOperator` once we have completed the transition and checked that the new weights work well?